app/models/comment.rb

```diff
@@ -246,6 +246,14 @@ def self.new_comment_from_email(mail)
         end
       end
       return node_id
+    else
+      email = mail.select { |s| s.match(/.*@.*/) }
+      ActionMailer::Base.mail(
+        from: "me@example.com", # @jywarren what should this be?
+        to: email,
+        subject: "Could not post your reply",
+        body: "Your reply wasn't posted since we couldn't find your account on publiclab.org. Please sign up on publiclab.org"
```

Comment:
Thank you! However, I'm not sure about this one, would it potentially attract spammers? Or, is this a good opportunity to recruit new users? But, how would they know what address to email if they aren't already a user? Thanks for thinking on this!!

Reply:
Hi @jywarren. Even I'm not too sure how they would get our email ID.
Going by this [comment](https://github.com/publiclab/plots2/issues/3322#issuecomment-472016501), I thought people reply to the email we send them...but again I'm not sure how we know their email ID if they're not a user.
Please let me know if I'm missing out on something. Thanks!

Reply:
OK I think this makes sense. I appreciate it!
```suggestion
        body: "Your reply wasn't posted since we couldn't find your account on publiclab.org. Please sign up at https://publiclab.org or try sending email from the account matching your username. Thank you!"
```